nixos/modules/programs/regreet.nix

```
@@ -58,9 +58,100 @@ in
         modifiable properties.
       '';
     };
+
+    theme = {
+      package = lib.mkOption {
+        type = lib.types.package;
+        default = pkgs.gnome-themes-extra;
+        defaultText = lib.literalExpression "pkgs.gnome-themes-extra";
+        description = ''
+          The package that provides the theme given in the name option.
+        '';
+      };
```

```suggestion
    theme = {
      package = lib.mkPackageOption pkgs "gnome-themes-extra" { } // {
        description = ''
          The package that provides the theme given in the name option.
        '';
      };
```
Package options should use `mkPackageOption`.

---

Fixed :+1